packages/utilities/src/constants/chromosomes.js

```diff
@@ -1,5 +1,5 @@
-export const HUMAN_AUTOSOMES = Array.from(new Array(22), (x, i) => `chr${i + 1}`)
+export const HUMAN_AUTOSOMES = Array.from(new Array(22), (x, i) => `${i + 1}`)
```

Comment:
The ManhattanPlot component was the only place these constants were used, so I removed the `chr` to match the example data.